nautobot/docs/additional-features/jobs.md

```diff
@@ -508,7 +513,7 @@ class MyJob(Job):
 ### Accessing User and Job Result
 
 +/- 2.0.0
-    The web request is no longer accessible to running jobs.
+    The `request` property has been changed to a celery request instead of a Django web request and no longer includes the information from the web request that initiated the Job. The `user` object is now available as `self.user` instead of `self.request.user`.
```

Comment:
```suggestion
    The `request` property has been changed to a Celery request instead of a Django web request and no longer includes the information from the web request that initiated the Job. The `user` object is now available as `self.user` instead of `self.request.user`.
```